addons/wowl/doc/components/dropdown.md

```diff
@@ -211,6 +211,17 @@ But you can go even further by extending them:
   </li>
   ```
 
+You can also make dropdown right aligned by passing 'o_dropdown_menu_right' in menuClass
```

Comment:
I personally think it has been put in the right documentation place (the **make it shine** section). But I would maybe add a bit of emphasis, like some bolding or italics.